ruby-signature.gemspec

```diff
@@ -42,4 +42,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rubocop"
   spec.add_development_dependency "rubocop-rubycw"
   spec.add_development_dependency "minitest-reporters", "~> 1.3.6"
+  spec.add_development_dependency "json", "~> 2.3.0"
```

Comment:
Okay, JSON is a gem... I think the best way is to put RBS in JSON gem, but I think it's okay to have the signatures here for now.